travel_platform/travel_app/views.py

```diff
@@ -3,10 +3,23 @@
 
 from django.http import Http404
 
+from django.http import JsonResponse
+
 from django.shortcuts import render
 
 from .models import Query
 
 
 def index(request):
     return render(request, 'travel_app/main.html')
+
+
+def coord(request):
+    """
+    View added for checking of interaction of map from landing page with 
+    server.
+    :param request: 
+    :return: 
+    """
+    data = 'Lviv'
```

Comment:
tried to do it. But, I have no idea, why request.GET doesn't extract data by the "lon", "lat" keys.